frontend/config/stakingPrograms/celo.ts

```diff
@@ -0,0 +1,36 @@
+import { Contract as MulticallContract } from 'ethers-multicall';
+
+import { STAKING_TOKEN_PROXY_ABI } from '@/abis/stakingTokenProxy';
+import { AgentType } from '@/enums/Agent';
+import { EvmChainId } from '@/enums/Chain';
+import { StakingProgramId } from '@/enums/StakingProgram';
+import { TokenSymbol } from '@/enums/Token';
+import { Address } from '@/types/Address';
+
+import { ACTIVITY_CHECKERS, ActivityCheckerType } from '../activityCheckers';
+import { StakingProgramMap } from '.';
+
+export const CELO_STAKING_PROGRAMS_CONTRACT_ADDRESSES: Record<string, Address> =
+  {
+    [StakingProgramId.MemeCeloAlpha2]:
+      '0xc653622FD75026a020995a1d8c8651316cBBc4dA', // TODO: celo
```

Comment:
```suggestion
      '0x95D12D193d466237Bc1E92a1a7756e4264f574AB',
```
https://govern.olas.network/contracts/0x00000000000000000000000095d12d193d466237bc1e92a1a7756e4264f574ab